cql-sdk.props

```diff
@@ -18,6 +18,7 @@
 		<PackageReadmeFile>README.md</PackageReadmeFile>
 		<PublishRepositoryUrl>true</PublishRepositoryUrl>
 		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<NoWarn>CA1311,CS8601,CS8602,CS8604</NoWarn>
```

Comment:
Do we really need to suppress CS8601, CS8602 and CS8604? They are pretty useful, right?  CA1311 used to be quite important to flag internationalization issues (the original codebase did not work with a Dutch locale for example).